BfA/SiegeOfBoralus/Trash.lua

```diff
@@ -551,3 +567,22 @@ end
 function mod:AshvaneInvaderDeath(args)
 	self:ClearNameplate(args.destGUID)
 end
+
+-- Riptide Shredder
+
+function mod:RiptideShredderEngaged(guid)
+	self:CDBar(257270, 14.5) -- Iron Ambush
+	self:Nameplate(257270, 14.5, guid) -- Iron Ambush
+end
+
+function mod:IronAmbush(args)
+	self:Message(args.spellId, "purple")
+	self:Nameplate(args.spellId, 25.5, args.sourceGUID)
+	self:PlaySound(args.spellId, "alarm")
+	self:CDBar(args.spellId, 25.5)
+end
```

Comment:
What's the reason behind adding this? For this season this ability was changed to be instant-cast and randomly targeted.

Generally if we're going to alert something it has to be so you can prepare or react. For an instant-cast spell that only does damage alerting the cast doesn't seem to accomplish either goal. I'd like to avoid "alert fatigue" from non-notable casts.

Reply:
Since the spell always targets the furthest member and can be deadly for some classes, the party memebers needs to shift their positions ahead of time according to when the spell cools down.
https://www.warcraftlogs.com/reports/1A7MnK3qpYWtkFLj#fight=last&type=deaths&ability=256866